trackpy/motion.py

```diff
@@ -184,7 +184,10 @@ def subtract_drift(traj, drift=None):
 
     if drift is None:
         drift = compute_drift(traj)
-    return traj.set_index('frame', drop=False).sub(drift, fill_value=0)
+    traj.set_index('frame', inplace=True, drop=False)
+    for col in drift.columns:
```

Comment:
This will cost some performance, as the subtraction is no long vectorized over columns. I think there is a way to do this using the new `DataFrame.assign` method, but that might require a pretty recent version of pandas. Thoughts?